packages/remix/src/generators/library/lib/normalize-options.ts

```diff
@@ -1,11 +1,8 @@
-import type { Tree } from '@nx/devkit';
-import { extractLayoutDirectory, getWorkspaceLayout, names } from '@nx/devkit';
-import { getImportPath } from '@nx/js/src/utils/get-import-path';
-import {
-  normalizeDirectory,
-  normalizeProjectName,
-} from '../../../utils/project';
-import type { NxRemixGeneratorSchema } from '../schema';
+import type {Tree} from '@nx/devkit';
+import {extractLayoutDirectory, getWorkspaceLayout, names} from '@nx/devkit';
+import {getImportPath} from '@nx/js/src/utils/get-import-path';
+import {normalizeDirectory, normalizeProjectName,} from '../../../utils/project';
+import type {NxRemixGeneratorSchema} from '../schema';
```

Comment:
nitpick: I've noticed some format changes in these PRs, like the lack of spaces around imports. I don't think there's a format check on this repo, could you run a format and see if it removes some of these format-only changes from the PR?

Reply:
that's a good point 🤔

I thought i had webstorm set up to run the format on file save. I'll run nx format instead. 

i may open a PR to add a pre-commit hook to run it? would that be useful?

Reply:
I've had the same trouble with WebStorm. :) A pre-commit hook would be nice, but probably low priority.